examples/apps/echo/client/echo_client.py

```diff
@@ -201,12 +201,10 @@ def Main(args=None):
 	# Add worker input data
 	wo_params.add_in_data(message)
 
-	# Sign work order
+	# Encrypt work order request hash
 	private_key = utility.generate_signing_keys()
```

Comment:
Do client need to generate private key if it's not used for signing

Reply:
Its not required, will delete it